### PR TITLE
Serial out temp & pH

### PR DESCRIPTION
**Pull request by eucalvo**

### CODE REVIEW

**Review by jgfoster — CHANGES_REQUESTED** [review]
It looks like we have a couple failing tests. You should pull from main to get the latest so you can observe the same test failures. To get rid of the RTC warning you could clear the output buffer just before running the test. Then, you need to add the date/time line to your expected output.

**Review by jgfoster — APPROVED** [review]
LGTM

**Comment by jgfoster** [comment]
Fixed #146.

File: test/TestTemperatureControl.cpp [hunk 1]
```diff
@@ -60,19 +62,23 @@ unittest(AfterIntervalAndWithinDelta) {
...
+  DateTime_TC january(2021, 1, 15, 1, 48, 24);
+  january.setAsCurrent();
+  state->serialPort[0].dataOut = "";  // the history of data written
   // chiller is initially off and goes on when needed
   assertEqual(TURN_SOLENOID_OFF, state->digitalPin[PIN]);
   chiller.setTargetTemperature(20);
   delay(31000);
   chiller.updateControl(20.05);
   assertEqual(TURN_SOLENOID_ON, state->digitalPin[PIN]);
+  assertEqual("2021-01-15 01:48:55\r\nchiller on after 31000 ms\r\n", state->serialPort[0].dataOut);
```
Comment (prestoncarman) [inline test/TestTemperatureControl.cpp:81]: We should also have a test print for when the chiller turns off.

File: test/TestTemperatureControl.cpp [hunk 2]
```diff
@@ -116,11 +122,13 @@ unittest(OutsideDelta) {
   GodmodeState* state = GODMODE();
   state->reset();
   Heater heater;
+  state->serialPort[0].dataOut = "";  // the history of data written
   // heater is initially off, then turns on
   assertEqual(TURN_SOLENOID_OFF, state->digitalPin[PIN]);
   heater.setTargetTemperature(20);
   heater.updateControl(19.95);
   assertEqual(TURN_SOLENOID_ON, state->digitalPin[PIN]);
+  assertEqual("2021-01-15 01:48:24\r\nheater on after 0 ms\r\n", state->serialPort[0].dataOut);
```
Comment (prestoncarman) [inline test/TestTemperatureControl.cpp:131]: We should also have a test print for when the heater turns off.

File: src/Devices/PHControl.h [hunk 3]
```diff
@@ -5,6 +5,7 @@ class PHControl {
   // Class variable
   static PHControl* _instance;
   // instance variables
+  unsigned long lastSwitchMS = 0;
```
Comment (prestoncarman) [inline src/Devices/PHControl.h:8]: This seems out of place since you are only changing temperature.

Remove this line and include it in the ph change.

**Review by prestoncarman — CHANGES_REQUESTED** [review]

**Review by prestoncarman — APPROVED** [review]
LGTM

**Review by jgfoster — APPROVED** [review]

**Pull request merged**